bw2io/ecoinvent.py

```diff
@@ -0,0 +1,260 @@
+import re
+import zipfile
+from collections import defaultdict
+from pathlib import Path
+
+import bw2data as bd
+import ecoinvent_interface as ei
+import openpyxl
+from ecoinvent_interface.core import SYSTEM_MODELS
```

Comment:
suggestion [documentation]: instruct users on using the enum to provide the system model 

Since the enum is already imported here, this would save users some headache when trying to figure out the right system model name, even if some great work was done to damerau_levenshtein guess/identify it.